src/components/Button/Button.tsx

```diff
@@ -7,19 +7,20 @@ interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
 	isLarge?: boolean;
 }
 
-const Button = ({
+const Button = forwardRef<HTMLButtonElement, Props>(({
 	hasOutline = false,
 	isPrimary = false,
 	isLarge = false,
 	children,
 	...rest
-}: Props) => (
+}, ref) => (
 	<button
+		ref={ref}
 		className={`button ${isPrimary ? 'primary' : ''} ${hasOutline ? 'outline' : ''} ${isLarge ? 'large' : 'small'}`}
```

Comment:
```suggestion
		className='button'
		data-size={isLarge ? 'large' : 'small'}
		data-outline={hasOutline}
		data-primary={isPrimary}
```

For those kinds of states, it is nicer to use [`data-*` attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/How_to/Use_data_attributes) instead of shoving everything into `class`es.

Reply:
@madcampos Should we make these changes after we complete the frontend integration as we will have to update buttons across all pages to align with these changes.

Reply:
No as it will not affact consumers of the component, only it's internals